src/network/networkd-link.c

```diff
@@ -721,10 +721,15 @@ void link_check_ready(Link *link) {
                     !link->ipv4ll_route)
                         return;
 
-        if (link_ipv6ll_enabled(link))
+        if (link_ipv6_enabled(link)) {
+
                 if (in_addr_is_null(AF_INET6, (const union in_addr_union*) &link->ipv6ll_address) > 0)
                         return;
 
+                if (link_ipv6_accept_ra_enabled(link) && !link->ndisc_configured)
```

Comment:
yes sure . There is something broken I guess.  Without https://github.com/systemd/systemd/commit/2b00a4e03dc375465de7f60f3a6937cbe8ffdf31 too bridge slave not able to get the IPv6 address even not receiving `RA` although client got started by networkd.